ui-core/src/main/java/com/adyen/checkout/components/extensions/FlowExtensions.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 30/11/2022.
+ */
+
+package com.adyen.checkout.components.extensions
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.flow.mergeStateFlows
+import com.adyen.checkout.components.ui.view.ComponentViewType
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.drop
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun mergeViewFlows(
```

Comment:
Not very important but is there any way we can have useful tests for this?

Reply:
Some cases are testable, but I'm not sure if we can simulate screen rotation somehow. Let me see what we can do

Reply:
I was only able to test the `drop` part and emitting of values

Reply:
That's good enough I think 🙏